### ISSUE TITLE
Panel monitoring for long-running tasks

### ISSUE THREAD

**Issue by shankiesan** [PixnBits/sublime-text-npm#29]
I'm using NPM as a basic build system in place of gulp/grunt – using the `scripts` property in package.json. I can run my scripts using `npm run` in the Sublime Package. This works, but it would be really nice if, for long-running processes (particularly 'watch' processes) we could see the results in a separate panel, similar to [Sublime Gulp](https://github.com/NicoSantangelo/sublime-gulp).

What do you think?

**Comment by PixnBits** [comment]
Yes! I've spent some effort on this, but I'm no Python guru :( Depending on their license, I may just redo this project based on Sublime Gulp, if they've figured out these hard parts :)

Similar to #24, #18 related

**Comment by shankiesan** [comment]
Indeed! I was looking to build something like this based on Sublime Gulp, realised I was a bit over my head with Python – so came here :smile: 

Yeah it's basically the same as #24 so I'll keep an eye on that.

Looks like Sublime Gulp is MIT, so you're in the clear!

If you need a hand, let me know, but my Python is the making-some-cool-things-happen-on-a-Raspberry-Pi sort of level.
